superset/utils/cache.py

```diff
@@ -55,7 +60,11 @@ def set_and_log_cache(
     if isinstance(cache_instance.cache, NullCache):
         return
 
-    timeout = cache_timeout if cache_timeout else config["CACHE_DEFAULT_TIMEOUT"]
+    timeout = (
+        cache_timeout
+        if cache_timeout
```

Comment:
```suggestion
        if cache_timeout is not None
```

I know this is just a rewrite of the above but `cache_timeout` could be `0` i.e., per Flask-Caching, 

> A timeout of 0 indicates that the cache never expires.

so a truthiness check isn't suffice.

Reply:
Oops!